types/shopify-buy/index.d.ts

```diff
@@ -95,7 +95,7 @@ declare namespace ShopifyBuy {
         /**
          * Update a line item quantity based on line item id
          */
-        updateLineItem(checkoutId: string | number, lineItems: AttributeInput[]): Promise<Cart>;
```

Comment:
![image](https://user-images.githubusercontent.com/25215475/102239477-e582d180-3ec4-11eb-89ff-378174eba12b.png)

My original error above. Are we sure this didn't get updated already? It looks like the name is being changed to `updateLineItems` plural again. Even though that was the original issue.